packages/mongodb-log-writer/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "@mongodb-js/mongodb-log-writer",
+  "description": "A library for writing MongoDB logv2 messages",
+  "version": "2.0.0",
```

Comment:
Heads up, especially since this is a major release, I'd include this in mongosh once this is done, then do a package release over there and update browser-repl & friends once that's done, so that all our products are on the 2.x version of this